class/defaults.yml

```diff
@@ -737,14 +741,20 @@ parameters:
           enabled: false
           restoreSA: mariadbrestoreserviceaccount
           restoreRoleRules: ${appcat:defaultRestoreRoleRules}
+          additionalInputs:
```

Comment:
maybe change from `additionalInputs` to `proxysql` ?

Reply:
We'd probably have to prefix it anyway to avoid naming clashes in the input object in the comp-function.